bin/update_tarball-install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+DEST=/usr/local/repo/tarball-install
+
+#############################
+### Mirror tarball-client ###
+#############################
+
+/usr/bin/lftp -c mirror                                              \
+  --delete                                                           \
+  --exclude-glob=osg-afs-client-*         `# Skip osg-afs-client`    \
+  --exclude-glob=*tarballs.rescue         `# Ignore rescue tarballs` \
+  --exclude-glob=osg-wn-client-latest.*   `# Ignore latest symlinks` \
+  --exclude=3.2                           `# Exclude old releases`   \
```

Comment:
Is only `3.2` considered old?  Whew.

Reply:
The production [repo.osg](http://repo.opensciencegrid.org/tarball-install/) goes back to 3.3. I have no opinion on how much history to retain.

Reply:
Neither do I  :)